apps/meteor/ee/client/voip/modal/DialPad/hooks/useDialPad.tsx

```diff
@@ -41,17 +48,32 @@ export const useDialPad = ({ initialValue, errorMessage }: DialPadProps): DialPa
 	});
 
 	const { ref, onChange } = register('PhoneInput');
+
 	const value = watch('PhoneInput');
 
 	const [disabled, setDisabled] = useState(true);
 
 	const handleBackspaceClick = useCallback((): void => {
+		clearErrors();
 		setValue('PhoneInput', value.slice(0, -1));
-	}, [setValue, value]);
+	}, [clearErrors, setValue, value]);
 
 	const handlePadButtonClick = useCallback(
-		(digit: string | number): void => {
-			setValue('PhoneInput', value + digit);
+		(digit: PadDigit): void => {
+			clearErrors();
+
+			setValue('PhoneInput', value + digit[0]);
+		},
+		[clearErrors, setValue, value],
+	);
+
+	const handlePadButtonLongPressed = useCallback(
+		(digit: PadDigit): void => {
+			if (digit[0] === '0') {
+				return setValue('PhoneInput', value + digit[1]);
+			}
+
+			setValue('PhoneInput', value + digit[0]);
```

Comment:
After the changes I recommended earlier, here you can just ignore digits different than `0`.
```
if (longDigit !== '0') {
  return;
}

// Do what you have to do
```

Reply:
ok!